.env

```diff
@@ -2,3 +2,7 @@ REACT_APP_VX_ENABLE_WRITE_IN_ADJUDICATION=TRUE
 REACT_APP_VX_ENABLE_ALL_ZERO_SMARTCARD_PIN_GENERATION=TRUE
 REACT_APP_VX_DISABLE_CARD_READER_CHECK=FALSE
 REACT_APP_VX_ENABLE_LIVECHECK=TRUE
+REACT_APP_VX_DISALLOW_CASTING_OVERVOTES=FALSE
```

Comment:
this is a huge nitpick but would it be possible to make this `ALLOW_CASTING_OVERVOTES` so it wouldn't be a double negative?

Reply:
out of scope of this pr